src/js/media-time-range.js

```diff
@@ -235,6 +236,11 @@ class MediaTimeRange extends MediaChromeRange {
     return attrVal != null ? +attrVal : undefined;
   }
 
+  get mediaPlaybackRate() {
+    const attrVal = this.getAttribute(MediaUIAttributes.MEDIA_PLAYBACK_RATE);
```

Comment:
nit(non-blocking): We can use "new-fangled" js (should probably start using this everywhere):
```js
return this.getAttribute(MediaUIAttributes.MEDIA_PLAYBACK_RATE) ?? 1;
````